kibbeh/src/stories/MobileHeader/ProfileHeader.story.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import { Story } from "@storybook/react";
+import {
+  ProfileHeader,
+  ProfileHeaderProps,
+} from "../../ui/mobile/MobileHeader/ProfileHeader";
+
+import src from "../../img/avatar.png";
+
+export default {
+  title: "MobileHeader/ProfileHeader",
+  component: ProfileHeader,
+};
+
+const TheProfileHeader: Story<ProfileHeaderProps> = ({
+  avatar = src,
+  onAnnouncementsClick = () => null,
+  onMessagesClick = () => null,
+  onSearchClick = () => null,
+}) => (
+  <div style={{ width: 420 }}>
```

Comment:
I think you don't need to set this width in the story container, there is a "small/large mobile" view setting in storybook.